spec/stack_spec.rb

```diff
@@ -228,6 +228,36 @@
     stack.delete
   end
 
+  it "return list of stacks" do
+    class ResponseMockStack
+      attr_reader :name
+
+      def initialize(name)
+        @name = name
+      end
+
+      def stack_name
+        return @name
+      end
+    end
+
+    class ResponseMock
+      def stack_summaries
+        return [ResponseMockStack.new("first"), ResponseMockStack.new("second")]
+      end
+    end
+
+    client_double = double
+    response = [ResponseMock.new, ResponseMock.new]
+    allow(Aws::CloudFormation::Client).to receive(:new).and_return(client_double)
+    allow(client_double).to receive(:list_stacks).and_return(response)
```

Comment:
Client specs should be making real AWS calls that are VCR recorded.  I can help with this later.

Reply:
I've updated this test, but I think I should be writing `VCR.use_cassette...` and use instead some global config file?